[Bug] [flinksqlenv] flinksqlenv task cannot check

### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dinky/issues?q=is%3Aissue) and found no similar issues.


### What happened

flinksqlenv任务不能检查。点击检查会出错。
![image](https://github.com/DataLinkDC/dinky/assets/123344357/f38f9192-b44a-4f27-a70c-a767e994b165)

报错如下：
```
Caused by: java.lang.RuntimeException: Not support dialect: FlinkSqlEnv
	at org.dinky.service.task.BaseTask.getTask(BaseTask.java:78) ~[dinky-admin-1.0.0-rc2.jar:?]
	at org.dinky.service.impl.TaskServiceImpl.explainTask(TaskServiceImpl.java:418) ~[dinky-admin-1.0.0-rc2.jar:?]
	at org.dinky.service.impl.TaskServiceImpl$$FastClassBySpringCGLIB$$22087f7c.invoke(<generated>) ~[dinky-admin-1.0.0-rc2.jar:?]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) ~[spring-core-5.3.27.jar:5.3.27]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:793) ~[spring-aop-5.3.27.jar:5.3.27]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-5.3.27.jar:5.3.27]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:763) ~[spring-aop-5.3.27.jar:5.3.27]
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:89) ~[spring-aop-5.3.27.jar:5.3.27]
	at org.dinky.aop.UdfClassLoaderAspect.round(UdfClassLoaderAspect.java:55) ~[dinky-admin-1.0.0-rc2.jar:?]
	... 91 more
```

### What you expected to happen

正常使用

### How to reproduce

none

### Anything else

_No response_

### Version

1.0.0

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dinky-admin/src/main/java/org/dinky/service/task/FlinkSqlEnvTask.java
+++ b/dinky-admin/src/main/java/org/dinky/service/task/FlinkSqlEnvTask.java
@@ -1,0 +1,75 @@
+/*
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.dinky.service.task;
+
+import org.dinky.assertion.Asserts;
+import org.dinky.config.Dialect;
+import org.dinky.data.annotations.SupportDialect;
+import org.dinky.data.dto.TaskDTO;
+import org.dinky.data.exception.NotSupportExecuteExcepition;
+import org.dinky.data.exception.NotSupportExplainExcepition;
+import org.dinky.data.result.SqlExplainResult;
+import org.dinky.gateway.enums.GatewayType;
+import org.dinky.job.JobManager;
+import org.dinky.job.JobResult;
+import org.dinky.service.impl.TaskServiceImpl;
+
+import java.util.List;
+
+import cn.hutool.core.text.StrFormatter;
+import cn.hutool.extra.spring.SpringUtil;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@SupportDialect(Dialect.FLINK_SQL_ENV)
+public class FlinkSqlEnvTask extends BaseTask {
+
+    protected final JobManager jobManager;
+
+    public FlinkSqlEnvTask(TaskDTO task) {
+        super(task);
+        // Default run mode is local.
+        if (Asserts.isNull(task.getType())) {
+            task.setType(GatewayType.LOCAL.getLongValue());
+        }
+        this.jobManager = getJobManager();
+    }
+
+    @Override
+    public List<SqlExplainResult> explain() throws NotSupportExplainExcepition {
+        return jobManager.explainSql(task.getStatement()).getSqlExplainResults();
+    }
+
+    @Override
+    public JobResult execute() throws Exception {
+        throw new NotSupportExecuteExcepition(StrFormatter.format(
+                "Task [{}] dialect [{}] don't support execute.", task.getName(), task.getDialect()));
+    }
+
+    @Override
+    public boolean stop() {
+        return false;
+    }
+
+    protected JobManager getJobManager() {
+        TaskServiceImpl taskService = SpringUtil.getBean(TaskServiceImpl.class);
+        return JobManager.build(taskService.buildJobSubmitConfig(task));
+    }
+}

--- a/dinky-common/src/main/java/org/dinky/data/exception/NotSupportExecuteExcepition.java
+++ b/dinky-common/src/main/java/org/dinky/data/exception/NotSupportExecuteExcepition.java
@@ -1,0 +1,26 @@
+/*
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.dinky.data.exception;
+
+public class NotSupportExecuteExcepition extends SqlExplainExcepition {
+    public NotSupportExecuteExcepition(String message) {
+        super(message);
+    }
+}

